.github/workflows/documentation.yml

```diff
@@ -16,21 +37,53 @@ jobs:
           default: true
           ldproxy: false
 
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          repository: mabezdev/esp-hal # TODO change once tagged in esp-rs
```

Comment:
In my repo I tagged the latest esp-wifi release as `esp-wifi-v0.9.1`

Reply:
This release has already been tagged